src/lib.rs

```diff
@@ -1,3 +1,114 @@
+//! # Floem
+//! Floem is cross-platform GUI framework for Rust 🦀. It aims to be extremely performant while providing world-class developer ergonomics.
+//!
+//! ## Views
+//! Floem models the UI using a tree of [Views](view::View) that are built once. `Views` react to state changes and events.
+//! Views ar self-contained components that can be composed together to create complex UIs.
+//!
+//! ## Ids and Id paths
```

Comment:
I don't think most developers using floem are going to need to understand ids. Especially not at first. I personally wouldn't add ID's to the top module documentation